src/routes/posts.rs

```diff
@@ -206,9 +212,13 @@ fn update(blog: String, slug: String, user: User, conn: DbConn, data: LenientFor
             post.content = SafeString::new(&content);
             post.source = form.content.clone();
             post.license = license;
+            println!("Cover id from update: {:?}", form.cover);
+            post.cover_id = form.cover;
             post.update(&*conn);
+            println!("Cover id after update: {:?}", post.cover_id);
             let post = post.update_ap_url(&*conn);
 
+            println!("Cover id after after update: {:?}", post.cover_id);
```

Comment:
Those 3 `println!` were for debug only I guess, so they should be removed

Reply:
Oops, you're right. :grimacing: